src/main.js

```diff
@@ -177,70 +177,212 @@ let destinations = [
+        photo: require('./img/santorini.jpg'),
         visited: false
     },
     {
         city: 'New York',
         country: 'U.S.',
-        photo: 'img/newyork.jpg',
+        photo: require('./img/newyork.jpg'),
         visited: false
     },
     {
         city: 'Marrakech',
         country: 'Morocco',
-        photo: 'img/marrakech.jpg',
+        photo: require('./img/marrakech.jpg'),
         visited: false
     },
     {
         city: 'Amsterdam',
         country: 'Netherlands',
-        photo: 'img/Amsterdam.png',
+        photo: require('./img/Amsterdam.png'),
         visited: false
     },
     {
         city: 'Berlin',
         country: 'Germany',
-        photo: 'img/berlin.jpg',
+        photo: require('./img/berlin.jpg'),
         visited: false
     },
     {
         city: 'Lisbon',
         country: 'Porugal',
-        photo: 'img/lisbon.jpg',
+        photo: require('./img/lisbon.jpg'),
         visited: false
     },
     {
         city: 'Tokyo',
         country: 'Japan',
-        photo: 'img/tokyo.jpg',
+        photo: require('./img/tokyo.jpg'),
         visited: false
     },
     {
         city: 'Sienna',
         country: 'Italy',
-        photo: 'img/nice.jpg',
+        photo: require('./img/sienna.jpg'),
         visited: false
 
     },
     {
         city: 'Dublin',
         country: 'Ireland',
-        photo: 'img/dublin.jpg',
+        photo: require('./img/dublin.jpg'),
         visited: false
     },
     {
         city: 'Saint petersburg',
         country: 'Russia',
-        photo: 'img/saintpetersburg.jpg',
+        photo: require('./img/saintpetersburg.jpg'),
         visited: false
     },
   ]
 
+ 
+
+  //get list of urls
+  const getURL = (destinationsList) => {
+    let url= destinationsList.map(obj => obj.photo.valueOf());
+    return url;
+  }
+
+  //create elements for each city
+  destinationsList.forEach(destination => {
+
+    //img wrapper
+    const createImg = (url) => {
+      let figure = document.createElement('figure');
+      figure.classList.add('img-container');
+    //img element
+      let img = document.createElement('img');
+      img.style.objectFit ='cover';
+      document.querySelector('.gallery').appendChild(figure);
+      figure.appendChild(img);
+    //img caption
+      let figcaption= document.createElement('figcaption');
+      figcaption.classList.add('caption');
+      figure.appendChild(figcaption);
+      figcaption.insertAdjacentHTML("beforeend", `<h2 class='caption__city'>${destination.city}</h2>` );
+      figcaption.insertAdjacentHTML("beforeend", `<h3 class='caption__country'>${destination.country}</h3>` )
+
+    //urls
+      img.src = destination.photo
+      
+    //add classes 
+      if(destination.visited === true) {
+        img.classList.add('visited');
+        figcaption.classList.add('visited');
+
+      }
+      else {
+        img.classList.add('bucketlist');
+        figcaption.classList.add('bucketlist');
+
+      }
```

Comment:
To simplify you could add `bucketlist` to `figcaption` instead. It would simplify the code and the CSS.

Reply:
Not sure what you mean, isn't this adding ```bucketlist``` to ```figcaption```?

``` figcaption.classList.add('bucketlist'); ```

Reply:
Sorry my bad, I meant to add it the parent, `figure`